cypress/components/popover-container/popover-container.cy.tsx

```diff
@@ -200,32 +200,6 @@ context("Test for Popover Container component", () => {
       popoverContainerContent().should("be.visible");
     });
 
-    it.each([
```

Comment:
**comment**: I'm not a fan of just completely removing tests provided they were there for good reason, which it seems like this one is (it's testing that a prop works as expected).

If the test was flaky I'd rather fix that up if we easily can - and if not then at least leave the test there in an `it.skip` with a comment explaining why it's been skipped and a note to investigate this in the future. (In this case I assume we could leave it till we convert the tests to Playwright.)

Reply:
I deleted these tests as they were flaky. They were relying on getting the DOMRect for the position and then asserted if that covered the button. Ideally, I could add these back in and skip them until we refactor to Playwright. As we'd be able to assert `.not.toBeInViewport` which is a lot neater than this in Cypress. I'll caveat this with we do also have a Chromatic snapshot of a story with the button covered, so it isn't like removing this test means we aren't covered for this case at all.

Reply:
Agree with the above, I'd prefer to have it `skip`ped rather than commented out though as even with the TODO it may be seen as something someone meant to remove etc